client/viame-web-common/components/AttributesSubsection.vue

```diff
@@ -0,0 +1,314 @@
+<script lang="ts">
+import {
+  defineComponent,
+  ref,
+  PropType,
+  computed,
+} from '@vue/composition-api';
+import { Attribute } from 'viame-web-common/apispec';
+import {
+  useSelectedTrackId,
+  useFrame,
+  useTrackMap,
+} from 'vue-media-annotator/provides';
+import Track, { TrackId } from 'vue-media-annotator/track';
+import AttributeInput from 'viame-web-common/components/AttributeInput.vue';
+
+
+function getTrack(
+  trackMap: Readonly<Map<TrackId, Track>>,
+  trackId: TrackId,
+): Track {
+  const track = trackMap.get(trackId);
+  if (track === undefined) {
+    throw new Error(`Track ${trackId} missing from map`);
+  }
+  return track;
+}
```

Comment:
should make this a util or something...

Reply:
Maybe it could be hoised from useTrackStore

Reply:
I added it into the provides and changed references to where we were using getTrack function or specifically using trackMap for only the purposes of calling getTrack.